app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -70,6 +74,24 @@ open class FirefoxApplication : LocaleAwareApplication() {
                 updateExperiments()
             }
 
+            // This is needed for Rust components to make HTTP network requests
+            Megazord.init()
+            RustHttpConfig.setClient(lazy { HttpUrlConnectionWrapper.client })
+
+            var admAvailable = false
```

Comment:
nit: We can probably re-introduce that setup method for push now that there are more checks that need to be done. 😅